src/truncate/truncate.html

```diff
@@ -0,0 +1,13 @@
+<span>
+  <span ng-if="!$ctrl.expanded && ctrl.truncated">
+    <span ng-attr-title="{{$ctrl.content}}" class="">
+      <span ng-bind-html="$ctrl.truncatedContent"></span>&hellip;
+    </span>
+    <a ng-href="" ng-click="$ctrl.expanded = true" class="nowrap">{{$ctrl.expandCopy||'See All'}}</a>
```

Comment:
Class currently is not defined, part of the original port. Clean up still required